torch/csrc/jit/cuda_fuser/interface.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+#include <atomic>
+#include <torch/csrc/WindowsTorchApiMacro.h>
+namespace torch {
+namespace jit{
+TORCH_API std::atomic<bool>& getCudaFusionGroupOptimizationPassMode();
+TORCH_API std::atomic<bool>& getCudaFusionGroupExecutorMode();
```

Comment:
is this going to be the switch between new fuser and old fuser on CUDA? it's just not super clear what bool value means. Maybe call it CudaFusionV2 or something?

Reply:
Do we want to rename the function to `getCudaFusionV2`?
The two switches controls: 1. whether we run the cuda fusion optimization pass; 2. whether we execution the new fusion node through the new codegen/executor.

The current naming scheme is a copy cat from our executor knobs: https://github.com/pytorch/pytorch/blob/d971007c291c0ead1003d12cd553d18ddb582207/torch/csrc/jit/graph_executor.h#L76-L77

I'll be happy to update the function name. But I'm not sure what is the suggested name for each function here.

Reply:
Since the knobs are removed. This is not relevant any more.